WalletWasabi.Fluent/Behaviors/ShowAttachedFlyoutWhenFocusedBehavior.cs

```diff
@@ -0,0 +1,126 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Diagnostics;
+using Avalonia.Controls.Primitives;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using Avalonia.VisualTree;
+using Avalonia.Xaml.Interactivity;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.Helpers;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class ShowAttachedFlyoutWhenFocusedBehavior : Behavior<Control>
```

Comment:
Unfortunately, using `OnAttached` doesn't give the desired results. 
I needed `OnAttachedToVisualTree` because otherwise `AssociatedObject.GetVisualRoot()` will return null. OnAttached is called before the visual root has been set.

```
		if (AssociatedObject.GetVisualRoot() is not Control visualRoot)
		{
			return;
		}
```

Reply:
Thanks! I didn't know this one existed. Handy :)